image.yaml

```diff
@@ -31,7 +31,7 @@ modules:
           - name: jboss-eap-7-image
             git:
                 url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                ref: EAP_726_CR1
+                ref: EAP_727_CR3
```

Comment:
@drichtarik Maybe use `EAP_727_1` instead of `EAP_727_CR3` (the former is newer -- 2 days old, while the latter is 21 days old) @hmlnarik @rgdoliveira @luck3y What's your take on this -^?

Reply:
+1, seems that the _1 contains the cve fix

Reply:
+2, EAP_727_1 is the correct tag.